test/testdrive/catalog.td

```diff
@@ -419,8 +419,7 @@ $ postgres-execute connection=postgres://mz_system@${testdrive.materialize-inter
 ALTER SYSTEM SET enable_rbac_checks TO true
 
 # Schema names that start with "mz_" or "pg_" are reserved for future use by the
-# system.
-! CREATE SCHEMA mz_foo
+# system. ! CREATE SCHEMA mz_foo
```

Comment:
This looks like an accidental but load bearing rewrap!

Reply:
eeek! nice catch.